runtime/moonriver/tests/integration_test.rs

```diff
@@ -173,7 +185,8 @@ fn verify_pallet_indices() {
 	// Handy utilities
 	is_pallet_index::<moonriver_runtime::Utility>(30);
 	is_pallet_index::<moonriver_runtime::Proxy>(31);
-	// Sudo was previously index 40
+	is_pallet_index::<moonriver_runtime::MaintenanceMode>(32);
+	// TODO Sudo was previously index 40, should we test that there is nothing there now?
```

Comment:
Is it important that nothing else claims this previously-used slot? If so, that sounds like a good idea.

Reply:
The whole point of manually assigning and testing pallet indices (instead of letting them be sequential and floating around is to maintain as much backwards compatibility of serialized transactions as possible.

For example if we had not supplied fixed indices, then when we removed sudo, all pallets listed after it would have had their indices change. That is not a problem for tools like polkadot js that dynamically construct their transactions with help from the node metadata. But it is a problem for tools that are more static like hardware wallets.

Making sure that nothing new ends up at a previously-used slot is a more minor example of the same problem. If we put a new pallet at slot 40, an old transaction constructing tool, may construct what it thinks is a sudo transaction, but the chain interprets differently (according to whatever pallet is there now). Most likely the sudo transaction would be invalid, but there is a chance it could actually execute some entirely different logic. So that's why it would be nice to have the test, but it is definitely a minor edge case. I don't want to block this PR on that.